README.md

```diff
@@ -462,13 +460,64 @@ Additionally, the OIDC `redirect_uri` parameter must be set to the HTTP API's au
 This is required, as the Authorization Code Grant needs to be redirected back to the HTTP API, to complete
 HTTP API token generation.
 
-### Add your specified `irods_user_claim` to the user's claims
+### Configure your 'user_mapping'
```

Comment:
I'm thinking we should change this title. The use of `your` seems weird to me.

Alternative Titles _(in no particular order)_:
- Configuring your User Mapping
- Configuring the 'user_mapping'
- User Mapping
- Mapping Users
- Mapping Users to iRODS
- User Mapping Configuration

Reply:
perhaps...

Mapping OpenID Users to iRODS

Reply:
That works.

Reply:
Done.